server/app/services/databaseConnection/DBConnection.java

```diff
@@ -0,0 +1,109 @@
+package services.databaseConnection;
+
+import play.Configuration;
+import play.api.Play;
+
+import java.sql.*;
+import java.util.ArrayList;
+
+/**
+ * Created by sabinapokhrel on 11/7/16.
+ */
+
+
+public class DBConnection {
```

Comment:
If this database operation service is mainly about `questions`, please rename it as `QuestionsDao` for a Data Access Object.